awx/main/models/__init__.py

```diff
@@ -91,14 +91,13 @@ def enforce_bigint_pk_migration():
         'main_systemjobevent'
     ):
         with connection.cursor() as cursor:
-            try:
-                cursor.execute(f'SELECT MAX(id) FROM _old_{tblname}')
```

Comment:
postgres logs an error for this, and it's kind of noisy:

<img width="1110" alt="image" src="https://user-images.githubusercontent.com/214912/77985872-97e91900-72e3-11ea-9533-5e98edcbbb7f.png">

Reply:
@ghjm I added you to this review, because you pointed this out, and you were right 😄 

https://github.com/ansible/awx/pull/6032#discussion_r382809053